app/assets/javascripts/components/plant_population_submission.coffee

```diff
@@ -12,6 +12,14 @@ window.PlantPopulationSubmission = class PlantPopulationSubmission extends Submi
     @$('.population-type').select2(@defaultSelectOptions)
     @$('.plant-line-list').select2(@plantLineListSelectOptions)
 
+    fields = [
+      'establishing-organisation'
+    ]
+
+    $.each fields, (_, field) =>
+      @$(".#{field}").select2(@defaultSelectOptions)
+      @$(".#{field}-wrapper").comboField()
```

Comment:
@kammerer This is, strictly speaking, not a regression - this lack was present before. So I can let Piotr merge this PR and describe the problem in a separate issue. Tell me if you prefer it that way, or maybe it is something you are able to fix right away?